homeassistant/components/bluesound/services.py

```diff
@@ -2,67 +2,7 @@
 
 from __future__ import annotations
 
-from typing import NamedTuple
-
-import voluptuous as vol
-
-from homeassistant.const import ATTR_ENTITY_ID
-from homeassistant.core import HomeAssistant, ServiceCall
-from homeassistant.helpers import config_validation as cv
-
-from .const import ATTR_MASTER, DOMAIN
-
 SERVICE_CLEAR_TIMER = "clear_sleep_timer"
 SERVICE_JOIN = "join"
 SERVICE_SET_TIMER = "set_sleep_timer"
```

Comment:
Should we change the timer services to buttons? They are easier to use in the UI

Reply:
That's a good idea :+1: 

The course of action would be to implement the button entities and deprecate the services?

Reply:
Yep!